sonar-db/src/main/gen-java/org/sonar/db/protobuf/DbCommons.java

```diff
@@ -0,0 +1,818 @@
+    /**
+     * <code>optional int32 end_line = 2;</code>
+     *
+     * <pre>
+     * End line (inclusive). Absent means it is same as start line
+     * </pre>
+     */
+    public boolean hasEndLine() {
+      return ((bitField0_ & 0x00000002) == 0x00000002);
+    }
+    /**
+     * <code>optional int32 end_line = 2;</code>
+     *
+     * <pre>
+     * End line (inclusive). Absent means it is same as start line
+     * </pre>
+     */
+    public int getEndLine() {
+      return endLine_;
+    }
+
+    public static final int START_OFFSET_FIELD_NUMBER = 3;
+    private int startOffset_;
+    /**
+     * <code>optional int32 start_offset = 3;</code>
+     *
+     * <pre>
+     * If absent it means range starts at the first offset of start line
+     * </pre>
+     */
+    public boolean hasStartOffset() {
+      return ((bitField0_ & 0x00000004) == 0x00000004);
+    }
+    /**
+     * <code>optional int32 start_offset = 3;</code>
+     *
+     * <pre>
+     * If absent it means range starts at the first offset of start line
+     * </pre>
+     */
+    public int getStartOffset() {
+      return startOffset_;
+    }
+
+    public static final int END_OFFSET_FIELD_NUMBER = 4;
+    private int endOffset_;
+    /**
+     * <code>optional int32 end_offset = 4;</code>
+     *
+     * <pre>
+     * If absent it means range ends at the last offset of end line
+     * </pre>
+     */
+    public boolean hasEndOffset() {
+      return ((bitField0_ & 0x00000008) == 0x00000008);
+    }
+    /**
+     * <code>optional int32 end_offset = 4;</code>
+     *
+     * <pre>
+     * If absent it means range ends at the last offset of end line
+     * </pre>
+     */
+    public int getEndOffset() {
+      return endOffset_;
+    }
+
+    private void initFields() {
+      startLine_ = 0;
+      endLine_ = 0;
+      startOffset_ = 0;
+      endOffset_ = 0;
+    }
+    private byte memoizedIsInitialized = -1;
+    public final boolean isInitialized() {
+      byte isInitialized = memoizedIsInitialized;
+      if (isInitialized == 1) return true;
+      if (isInitialized == 0) return false;
+
+      memoizedIsInitialized = 1;
+      return true;
+    }
+
+    public void writeTo(com.google.protobuf.CodedOutputStream output)
+                        throws java.io.IOException {
+      getSerializedSize();
+      if (((bitField0_ & 0x00000001) == 0x00000001)) {
+        output.writeInt32(1, startLine_);
+      }
+      if (((bitField0_ & 0x00000002) == 0x00000002)) {
+        output.writeInt32(2, endLine_);
+      }
+      if (((bitField0_ & 0x00000004) == 0x00000004)) {
+        output.writeInt32(3, startOffset_);
+      }
+      if (((bitField0_ & 0x00000008) == 0x00000008)) {
+        output.writeInt32(4, endOffset_);
+      }
+      getUnknownFields().writeTo(output);
+    }
+
+    private int memoizedSerializedSize = -1;
+    public int getSerializedSize() {
+      int size = memoizedSerializedSize;
+      if (size != -1) return size;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Missing curly brace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00121)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00122)